### DISCUSSION TITLE
VSCode GraphQL Config not working with TypeScript 4.9

### DISCUSSION THREAD

**Discussion by nandorojo** [dotansimha/graphql-code-generator#9066] [Q&A]
Hey there, I know this maybe isn't the best place to ask, but I was hoping someone might know the answer.

I'm trying to use the `client-preset` codegen solution. That works fine with the codegen CLI. However, my IDE seems unable to parse the graphql strings in my .tsx file whenever I use `satisfies` from TypeScript.

```tsx
const a = { hi: true } satisfies { hi: boolean }

const query = graphql(`query SomeQuery { hi } }`) 
```

The result is this output on VSCode: 

```tsx
2/24/2023, 4:09:08 PM [1] (pid: 57176) graphql-language-service-usage-logs: SyntaxError: Missing semicolon. (625:15)
```

This means that I don't get autocomplete in that file.

Any ideas for what could fix this? Is there a way to upgrade the graphql-language-service to a version that's compatible with TypeScript 4.9?

Thanks!

**Comment by fabis94** [comment]
Having the same issue...
